container/container.go

```diff
@@ -124,7 +124,7 @@ func %v(writer io.Writer, codec Codec, recordsPerBlock int64) (*%v, error) {
 `
 
 const containerWriterWriteTemplate = `
-func (avroWriter *%v) WriteRecord(record %v) error {
+func (avroWriter *%v) Write(record avroutil.Serializable) error {
```

Comment:
Per our conversation the other day, please keep method names consistent with gogen-avro